search/client.go

```diff
@@ -76,3 +77,22 @@ func (c *client) Query(q Query) (*Response, error) {
 	err = json.NewDecoder(resp.Body).Decode(queryResponse)
 	return queryResponse, err
 }
+
+func (c *client) ListFacetValues(field string, maximumNumberOfValues int) (*FacetValues, error) {
+	req, err := http.NewRequest("GET", c.endpoint + "values?field=" + field + "&maximumNumberOfValues=" + strconv.Itoa(maximumNumberOfValues), nil)
```

Comment:
you should use `net/url` to construct the url instead of concatenating strings 

https://godoc.org/net/url